feat: add a seed value to test runs (#13400)
facebook/jest: docs/JestObjectAPI.md

## Misc

> ### `jest.getSeed()`
> Every time Jest runs a seed value is randomly generated which you could use in a pseudorandom number generator or anywhere else.

Reviewer: ```suggestion
### `jest.getSeed()`

Every time Jest runs a seed value is randomly generated. It can be used in a pseudorandom number generator or anywhere else.
```

:::tip

> You can manually define this value with the CLI argument `--seed=<num>`.

Reviewer: ```suggestion
You can manually set this value via the [`--seed=<num>`](CLI.md#seednum) CLI argument.
```

## Misc

### `jest.getSeed()`

Every time Jest runs a seed value is randomly generated which you could use in a pseudorandom number generator or anywhere else.

:::tip

> You can manually set this value via the [`--seed=<num>`](CLI.md#--seednum) CLI argument.

Reviewer: ```suggestion
Use the [`--showSeed`](CLI.md#--showseed) flag to print the seed in the test report summary. To manually set the value of the seed use [`--seed=<num>`](CLI.md#--seednum) CLI argument.
```
